pkg/s3/s3.go

```diff
@@ -0,0 +1,93 @@
+package s3
+
+import (
+	"fmt"
+	"math/rand"
+	"reflect"
+)
+
+const charset = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
+
+type BucketAccess struct {
```

Comment:
BucketAccess is what COSI uses, but S3 creates users. In the interest of showing users how COSI resources correlate to real-world object stores, I think it might make the most sense to name this `User` instead.

Reply:
Fixed.